src/tscore/unit_tests/test_Extendible.cc

```diff
@@ -209,20 +209,36 @@ TEST_CASE("Extendible", "")
     CHECK(testField::alive == 0);
   }
 
+  INFO("AcidPtr AcidCommitPtr malloc ptr int");
+  {
+    void *mem            = malloc(sizeof(AcidPtr<int>));
+    AcidPtr<int> &reader = *(new (mem) AcidPtr<int>);
+    {
+      auto writer = reader.startCommit();
+      CHECK(*writer == 0);
+      *writer = 1;
+      CHECK(*writer == 1);
+      CHECK(*reader.getPtr().get() == 0);
+      // end of scope writer, commit to reader
+    }
+    CHECK(*reader.getPtr().get() == 1);
```

Comment:
I suggest adding here:
`CHECK(*sp.get() == 0);`

Reply:
That is a good idea. I'll address that in a separate PR.